azurelinuxagent/ga/logcollector.py

```diff
@@ -326,7 +325,9 @@ def _get_final_list_for_archive(self, priority_file_queue):
                 if e.errno == 2:    # [Errno 2] No such file or directory
                     _LOGGER.warning("File %s does not exist, skipping collection for this file", file_path)
 
-        _LOGGER.info("Uncompressed archive size is %s b", total_uncompressed_size)
+        msg = "Uncompressed archive size is {0} b".format(total_uncompressed_size)
+        _LOGGER.info(msg)
+        add_event(op=WALAEventOperation.LogCollection, message=msg)
```

Comment:
Adding this event before the process enters the main loop in an effort to collect this info for runs that fail due to mem limit exceeded